src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -828,6 +828,12 @@ public void test_clean() throws Exception {
         w.touch(fileName2);
         w.touch(fileName, "new content");
 
+        // JGit does not support exclude patterns        
+        if (w.igit() instanceof CliGitAPIImpl)
+        {
+        	((CliGitAPIImpl)w.igit()).clean_().excludePatterns(Collections.singletonList(fileName)).execute();
```

Comment:
I was under the impression cgit/jgit methods return a new GitClient object. Opposed to the igit method which returns the current workspace GitClient. So the reasoning was to add an extra test for the CliGitAPIImplTest run using the workspace client. And skip the extra test for a JGitAPIImplTest run.